lib/shopify/session.ex

```diff
@@ -2,6 +2,7 @@ defmodule Shopify.Session do
   @moduledoc false
 
   alias Shopify.Config
+  alias Shopify.Session
```

Comment:
```elixir
# Do you prefer this
alias Shopify.Config
alias Shopify.Session

# over this?
alias Shopify.{Config, Session}
```

Was this an intentional choice? I usually prefer the `{}` notation, but I am willing to change my habbits 😃

Reply:
Fixed :+1 - just missed that